packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1475,21 +1200,354 @@ class _DialState extends State<_Dial> with SingleTickerProviderStateMixin {
         key: const ValueKey<String>('time-picker-dial'),
         painter: _DialPainter(
           selectedValue: selectedDialValue,
-          primaryOuterLabels: primaryOuterLabels,
-          primaryInnerLabels: primaryInnerLabels,
-          secondaryOuterLabels: secondaryOuterLabels,
-          secondaryInnerLabels: secondaryInnerLabels,
+          primaryLabels: primaryLabels,
+          secondaryLabels: secondaryLabels,
           backgroundColor: backgroundColor,
-          accentColor: themeData.accentColor,
+          accentColor: accentColor,
+          dotColor: theme.colorScheme.surface,
           theta: _theta.value,
-          activeRing: _activeRing,
           textDirection: Directionality.of(context),
         ),
       ),
     );
   }
 }
 
+class _TimePickerInput extends StatefulWidget {
+  const _TimePickerInput({
+    Key key,
+    @required this.initialSelectedTime,
+    @required this.helpText,
+    @required this.onChanged,
+  }) : assert(initialSelectedTime != null),
```

Comment:
Looks onChanged is assumed to be non-null, assert here.

Reply:
Done